graph-service/enricher.py

```
@@ -0,0 +1,875 @@
+        try:
+            # Enrich a batch of nodes
+            await enrich_batch(driver, browser, batch_size=10)
+        finally:
+            await browser.close()
+            if driver:
+                driver.close()
+
+def main():
+    # Schedule to run every 4 hours
+    schedule.every(4).hours.do(lambda: asyncio.run(run_enrichment()))
+    logger.info("Enricher started – scheduled to run every 4 hours")
```

Ruff (0.14.7)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)